lib/data/entity.js

```diff
@@ -86,7 +75,7 @@ const parseSubmissionXml = (entityFields, xml) => new Promise((resolve, reject)
   });
   stream.on('end', () => {
     try {
-      resolve(validateEntity(entity));
+      resolve(entity);
```

Comment:
this function now just parse the dataset/entity related information. Validation is done by consumer function